src/main/java/uk/gov/digital/ho/hocs/casework/domain/exception/RestResponseEntityExceptionHandler.java

```diff
@@ -75,7 +79,10 @@ public ResponseEntity handle(HttpMessageNotReadableException e) {
 
     @ExceptionHandler(Exception.class)
     public ResponseEntity handle(Exception e) {
-        log.error("Exception: {}", e.getMessage(), value(EVENT, UNCAUGHT_EXCEPTION));
+        Writer stackTraceWriter = new StringWriter();
+        PrintWriter printWriter = new PrintWriter(stackTraceWriter);
+        e.printStackTrace(printWriter);
```

Comment:
Does this get output as JSON? If not, it'll be a pain to dig out of Kibana

Reply:
The logback configuration should output the logs as json based on the profile used in dev.